clientv3/integration/kv_test.go

```diff
@@ -59,9 +59,9 @@ func TestKVPutError(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	time.Sleep(500 * time.Millisecond) // give enough time for commit
+	time.Sleep(1 * time.Second) // give enough time for commit
 
-	_, err = kv.Put(ctx, "foo2", strings.Repeat("a", int(maxReqBytes-50)))
+	_, err = kv.Put(ctx, "foo2", strings.Repeat("a", int(quota-50)))
```

Comment:
This is testing the over quota alarm, so in the general case, we should be using the quota size to test with.   If we have a quota size of 4MiB, then 1.5 MiB - 50 won't hit the over quota.